### ISSUE TITLE
Waveshare board issue?

### ISSUE THREAD

**Issue by yudhisteer** [AlexanderKoch-Koch/low_cost_robot#36]
Hi,
I bought two waveshare board and when testing the motors, it is not being recognized. What is the probability that two of the waveshare board is not good? 

![D1](https://github.com/AlexanderKoch-Koch/low_cost_robot/assets/59663734/b06dc38e-d60f-4cf8-ba42-e1fe45d597cd)

![image](https://github.com/AlexanderKoch-Koch/low_cost_robot/assets/59663734/78e2a397-441a-46b3-a2ef-0698acec1fb1)

**Comment by ex3ndr** [comment]
You need to supply power to them to power up

**Comment by MichaelrMentele** [comment]
I'm not seeing the motor either.
![IMG_1730](https://github.com/AlexanderKoch-Koch/low_cost_robot/assets/9519712/47712ee1-8f7c-46bf-82be-ff3b217a3bd0)

I do see the board:
<img width="819" alt="image" src="https://github.com/AlexanderKoch-Koch/low_cost_robot/assets/9519712/13a85eaa-a86b-4b14-8e78-38e1d169e553">

But scan finds nothing. Tried two boards. Swapped the motors.

**Comment by MichaelrMentele** [comment]
> Hi, I bought two waveshare board and when testing the motors, it is not being recognized. What is the probability that two of the waveshare board is not good?

It looks like you can detect the motor when doing the firmware updates though based on your screenshot?

**Comment by MichaelrMentele** [comment]
For me it's definitely the power supply. The 5v won't power the 330 for more than a moment. The 12 v causes the 330 to burn up but won't power the 430 for more than a moment.

**Comment by MichaelrMentele** [comment]
@yudhisteer any update?

**Comment by MichaelrMentele** [comment]
I had one bad board -- got a new one and it worked. Makes you want to just use the U2D2

**Comment by karanchahal** [comment]
Yeah for sure, U2D2 would have helped I think. But now I am in too deep, I burnt 2 waveshares in this process.
